### DISCUSSION TITLE
What stops collecting GPS coords / track recording ?

### DISCUSSION THREAD

**Discussion by printmaps** [devemux86/cruiser#657] [Q&A] [answered]
I have done this:

- activated 'map follow mode'
- started track recoding (at this point everything works as expected)
- after a while I stopped 'map follow mode' (but not track recording)
  - reason for this: I want to look for something on the map in north orientation 
  - at this point no further GPS points are recorded in the track
- after a while I activated 'map follow mode' again
- observations:
  - recorded track has a gap
  - line between last and current position not drawn on map 

Screenshot from Locus Map with correct track:
![Screenshot_20241026_154102_Locus Map Classic](https://github.com/user-attachments/assets/b2e3f14e-c650-4daf-8be3-989e80890829)

Screenshot from Cruiser with gap:
![Screenshot_20241026_153936_Cruiser](https://github.com/user-attachments/assets/b9d9c179-357d-4159-b347-edea27548451)

What was my expectation? In case of 'track recording' nothing should stop collecting GPS positions.

Question: Is the observed behavior a bug, a feature or a user fault?

**Comment by devemux86** [comment, marked as answer]
Do you have enabled the "Settings | GPS recording | Automatic recording of track" for "Navigation" or "Follow location"?
Then the GPS recording starts / stops **automatically** when the navigation or follow location starts / stops.
(it is disabled by default)

If you want to start / stop **manually** the GPS recording, then do not enable this setting.

(when GPS recording starts it creates a new gpx track segment)

<img src="https://github.com/user-attachments/assets/8579dd5b-30e3-4577-a77b-3730bea16e97" width="50%"/>

**Comment by printmaps** [comment]
Both settings are checked (marked as enabled) in my case. This explains the observed behavior ... but is confusing. Could it be overwritten (deactivated) in case that the user starts track recording manually?

**Reply by devemux86** [reply]
Usually the opposite is the case, when a setting is enabled it is expected to work.

They are disabled by default, so if users turn them on, they expect them  to work automatically.
